library/styleguideComponents/Wrapper/index.js

```diff
@@ -65,6 +71,8 @@ const cyanDarkTheme = {
             success: '#b9f6ca',
             error: '#ff5252',
             mainBackground: '#303030',
+            header: 'rgba(255, 255, 255, 0.64)',
+            badge: 'white',
```

Comment:
we should not add values that do not belong to the theme API

Reply:
Got it, updating that now